src/components/sliders/draw.js

```diff
@@ -71,14 +71,12 @@ module.exports = function draw(gd) {
 
         computeLabelSteps(sliderOpts);
 
-        if(!sliderOpts._commandObserver) {
-            sliderOpts._commandObserver = Plots.createCommandObserver(gd, sliderOpts.steps, function(data) {
-                if(sliderOpts.active === data.index) return;
-                if(sliderOpts._dragging) return;
+        Plots.createCommandObserver(gd, sliderOpts, sliderOpts.steps, function(data) {
```

Comment:
I'm big a fan of this new pattern here where one routine creates + updates command observer. This should make the command observer logic more easily reused in the future when we integrate it in other components :+1: 

One small suggestion, now that `createCommandObsever` can create and update a command observer, maybe we should rename it something like `manageCommandObserver` similar to [`manageModebar`](https://github.com/plotly/plotly.js/blob/ec0b0a49326f689dee01b228704dfb9ac1c9ea78/src/components/modebar/manage.js#L27)?